libcentrifugo/application.go

```diff
@@ -498,7 +498,7 @@ func (app *Application) pubPing() error {
 		Goroutines: runtime.NumGoroutine(),
 		NumCPU:     runtime.NumCPU(),
 		Gomaxprocs: runtime.GOMAXPROCS(-1),
-		Metrics:    *app.metrics,
+		Metrics:    *app.metrics.GetSnapshot(),
```

Comment:
Huh ok well go with this for now. Does it solve all races reported?

Sorry I didn't think to check this.